### ISSUE TITLE
grand mesa demo

### ISSUE THREAD

**Issue by slhowardESR** [ICESat2-SlideRule/sliderule-python#68]
I ran the grand mesa demo.

if failed on this line: 

```
# Query ATL06 Files from NASA CMR System
resources = icesat2.cmr(region, short_name='ATL06')

```
![Capture](https://user-images.githubusercontent.com/5480396/150197864-31ef8c48-4739-4505-87d5-acf80482e7e5.PNG)

I think it needs to be changed to :

`resources = icesat2.cmr(polygon=region, short_name='ATL06')    `

it worked fine for me after that.

btw, please let me know if there is a better place to put these minor issues.  (if i should put them on slack, etc).

**Comment by jpswinski** [comment]
@slhowardESR this is the perfect place to let us know about these issues, we appreciate you letting us know, and thank you for catching this issue.  I should be able to fix this very quickly - I'll let you know once the new version is pushed.

**Comment by jpswinski** [comment]
@slhowardESR this has been fixed in the latest version of the code just released (v1.3.3).  So if you do a pull you should be good to go.  I am going to close this issue now, but if you find in the future that this is still a problem, please feel free to reopen this issue or contact us directly.
